core/server/master/src/main/java/alluxio/master/file/meta/MountTable.java

```diff
@@ -253,15 +257,22 @@ public boolean delete(Supplier<JournalContext> journalContext, AlluxioURI uri) {
    */
   public String getMountPoint(AlluxioURI uri) throws InvalidPathException {
     String path = uri.getPath();
-
+    String candidatePath = ROOT;
     try (LockResource r = new LockResource(mReadLock)) {
       for (Map.Entry<String, MountInfo> entry : mState.getMountTable().entrySet()) {
         String alluxioPath = entry.getKey();
```

Comment:
NIT: can we rename this to `mount` or something? All the prefix checks are very confusing with the current names.

Reply:
done